app/admin/councillor_contributions.rb

```diff
@@ -21,7 +22,25 @@
     end
   end
 
-  action_item :download,:only => [:show] do
+  form title: "edit" do |f|
+    inputs "Councillor contribution" do
+      input :contributor
```

Comment:
Given that we have no need to edit the contributor or source at the moment, I think we shouldn't have these attributes available to edit.

Would another way to do this be to have just an action button on the admin page that says 'mark as reviewed' if it's not reviewed, and 'mark as not reviewed' if it's been reviewed so the admin can toggle it back if they need (I can imaging that being useful) ? Would that be simpler?